lib/rules/lint-no-arguments-for-html-elements.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Rule = require('./base');
+
+const ERROR_MESSAGE = 'Argument "%" won\'t work for HTML elements ("<%>").';
```

Comment:
Instead of creating this custom string interpolation scheme with the percent signs, you should just use and export functions that look like:

```
function makeErrorMessage(argumentName) {
  return `...${argumetName}`;
}
```